### PR TITLE
Use Fernet instead of raw primitive AES-CBC

### PR DESCRIPTION
**Pull request by grydz**
Use [Fernet](https://github.com/fernet/spec/blob/master/Spec.md) for more a realistic example and avoid pitfalls with raw AES-CBC primitive.

### CODE REVIEW

**Review by ThibsG — APPROVED** [review]

File: mse_home/template/secrets.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-    "key": "00112233445566778899aabbccddeeff"
+    "sk": "PVyaHlCyN40t5YeAgqLxbz8aISYg1QbRN4qyo_QqIlM="
```
Comment (heavenboy8) [inline mse_home/template/secrets.json]: "password"?

File: mse_home/template/secrets_to_seal.json [hunk 2]
```diff
@@ -1,3 +1,3 @@
 {
-    "key": "ffeeddccbbaa99887766554433221100"
+    "sealed_sk": "AXIKImqLJa5SZms7o6mb_nL1QLro_GDNcpIJQ71CgMk="
```
Comment (heavenboy8) [inline mse_home/template/secrets_to_seal.json]: "result_key"?

File: mse_home/template/secrets.json (hunk 1 above)
Comment (grydz) [reply mse_home/template/secrets.json]: Fixed in ffdad282e7d508216010f865077d045ddc712c2a.

File: mse_home/template/secrets_to_seal.json (hunk 2 above)
Comment (grydz) [reply mse_home/template/secrets_to_seal.json]: Fixed in ffdad282e7d508216010f865077d045ddc712c2a.

**Pull request merged**